client/my-sites/plugins/plugin-install-button/index.jsx

```diff
@@ -302,6 +342,7 @@ export default connect(
 		return {
 			siteId,
 			siteIsConnected: getSiteConnectionStatus( state, siteId ),
+			isWpcomAtomicSite: isSiteWpcomAtomic( state, siteId ),
```

Comment:
Similar to `siteIsConnected` on the line above, naming this `siteIsWpcomAtomic` would make the prop sound less like a function that answers a question.